tests/testthat/test-use-tutorial.R

```diff
@@ -20,3 +20,14 @@ test_that("use_tutorial() creates a tutorial", {
     }
   )
 })
+
+test_that("use_tutorial(learner_friendly = TRUE) adds promised file, Imports learnr", {
+  with_mock(
+    `usethis:::uses_roxygen` = function(base_path) TRUE, {
```

Comment:
This also needs to use a mock for  `usethis:::is_installed`, as the test above it does. See the test failures on travis and appveyor.